Dockerfile.lambda

```diff
@@ -0,0 +1,15 @@
+FROM public.ecr.aws/lambda/python:3.9-x86_64
```

Comment:
@rlkelly This `Dockerfile.lambda` is a small modification of  https://github.com/42labs/Empiric/blob/zksync_publisher_minor_refactor/stagecoach/jobs/publishers/evm-publisher/Dockerfile

Do we still need the latter `Dockerfile`?